integ/scaleup_test.go

```diff
@@ -1,151 +1,145 @@
-	c.Assert(err, IsNil)
-
-	body := res.(map[string]interface{})
-	restored := (body["latest"].(map[string]interface{}))["restored"]
-	c.Assert(restored, NotNil)
-
-	c.Assert(restored.(map[string]interface{})["is_savepoint"], Equals, true)
-
-	// check that we have the correct number of total pods
-	pods, err = s.Util.KubeClient.CoreV1().Pods(s.Util.Namespace.Name).
-		List(v1.ListOptions{LabelSelector: "integTest=" + testName})
-	c.Assert(err, IsNil)
-	c.Assert(len(pods.Items), Equals, 3)
-
-	// check that we are still using the same deploymnet
-	deployments2, err := s.Util.KubeClient.AppsV1().Deployments(s.Util.Namespace.Name).
-		List(v1.ListOptions{LabelSelector: "flink-app=inplace,flink-deployment-type=taskmanager"})
-	c.Assert(err, IsNil)
-	c.Assert(len(deployments2.Items), Equals, 1)
-	deployment2 := deployments.Items[0]
-	c.Assert(deployment2.Name, Equals, deployment.Name)
-
-	// ensure that we can now proceed to a normal deployment
-	newApp = updateAndValidate(c, s, config.Name, func(app *v1beta1.FlinkApplication) {
-		app.Spec.Image = NewImage
-	}, v1beta1.FlinkApplicationDeployFailed)
-	c.Assert(newApp.Spec.Image, Equals, NewImage)
-	pods, err = s.Util.KubeClient.CoreV1().Pods(s.Util.Namespace.Name).
-		List(v1.ListOptions{LabelSelector: "integTest=" + testName})
-	c.Assert(err, IsNil)
-	c.Assert(len(pods.Items), Equals, 3)
-	for _, pod := range pods.Items {
-		c.Assert(pod.Spec.Containers[0].Image, Equals, NewImage)
-	}
-
-	// delete the application and ensure everything is cleaned up successfully
-	c.Assert(s.Util.FlinkApps().Delete(config.Name, &v1.DeleteOptions{}), IsNil)
-
-	// validate that a savepoint was taken and the job was cancelled
-	var app *v1beta1.FlinkApplication
-	for {
-		app, err = s.Util.GetFlinkApplication(config.Name)
-		c.Assert(err, IsNil)
-
-		if len(app.Finalizers) == 1 && app.Finalizers[0] == finalizer {
-			break
-		}
-		time.Sleep(100 * time.Millisecond)
-	}
-
-	c.Assert(app.Status.SavepointPath, NotNil)
-	job := func() map[string]interface{} {
-		jobs, _ := s.Util.FlinkAPIGet(app, "/jobs")
-		jobMap := jobs.(map[string]interface{})
-		jobList := jobMap["jobs"].([]interface{})
-		for _, j := range jobList {
-			job := j.(map[string]interface{})
-			if job["id"] == app.Status.JobStatus.JobID {
-				return job
-			}
-		}
-		return nil
-	}()
-
-	fmt.Printf("test job = %v", job)
-	c.Assert(job["status"], Equals, "CANCELED")
-
-	// delete our finalizer
-	app.Finalizers = []string{}
-	_, err = s.Util.FlinkApps().Update(app)
-	c.Assert(err, IsNil)
-
-	// wait until all pods are gone
-	for {
-		pods, err = s.Util.KubeClient.CoreV1().Pods(s.Util.Namespace.Name).
-			List(v1.ListOptions{LabelSelector: "integTest=" + testName})
-		c.Assert(err, IsNil)
-		if len(pods.Items) == 0 {
-			break
-		}
-		time.Sleep(100 * time.Millisecond)
-	}
-	log.Info("All pods torn down")
-}
+// TODO: https://github.com/lyft/flinkk8soperator/issues/278
+//func (s *IntegSuite) TestInPlaceScaleUp(c *C) {
```

Comment:
Instead of commenting all this out I believe you can just do 
`c.Skip("Skipping because of blah")`